### PR TITLE
Link to blog

### PR DESCRIPTION
**Pull request by dstiberova**
Please review

### CODE REVIEW

File: resources/public/css/style.css [hunk 1]
```diff
@@ -554,4 +554,30 @@ input[type="submit"].btn {
         left: initial;
         top: initial;
     }
-}
\ No newline at end of file
+}
+
+.cooperation {text-align: center;
```
Comment (tomasz-biernacki) [inline resources/public/css/style.css]: I would align this better, just to be consistent with indentation.

File: resources/public/css/style.css [hunk 2]
```diff
@@ -554,4 +554,30 @@ input[type="submit"].btn {
...
+              display: inline-block;
+              vertical-align: top;
+              width: 33.333;}
+
+.cooperation p {margin-bottom: 0px;}
```
Comment (tomasz-biernacki) [inline resources/public/css/style.css]: Here, too as above.

File: resources/public/css/style.css [hunk 3]
```diff
@@ -554,4 +554,30 @@ input[type="submit"].btn {
...
+    padding-bottom: 5px;
+    padding-left: 25px;
+    padding-right: 25px;
+    color: white;
+    /* font-size: 13px */;
+}
+
+.blog-text {
+    margin-top: 0px;
+    padding-top: 0px;}
```
Comment (tomasz-biernacki) [inline resources/public/css/style.css]: And here too

**Comment by tomasz-biernacki** [comment]
It's good to squash if this is a final PR (although now it's possible in gh).
And also remember to check for mobile.

**Comment by dstiberova** [comment]
@tomasz-biernacki I checked it on mobile and it looks ok

**Comment by arichiardi** [comment]
Yes please let's change the indentation

**Comment by dstiberova** [comment]
@arichiardi @tomasz-biernacki indentation changed

File: resources/public/css/style.css [hunk 4]
```diff
@@ -556,17 +556,20 @@ input[type="submit"].btn {
...
 
-.cooperation {text-align: center;
-              display: inline-block;
-              vertical-align: top;
-              width: 33.333;}
+.cooperation {
+    text-align: center;
+    display: inline-block;
+    vertical-align: top;
+    width: 33.333;}
```
Comment (tomasz-biernacki) [inline resources/public/css/style.css]: newline ;)

File: resources/public/css/style.css [hunk 5]
```diff
@@ -556,17 +556,20 @@ input[type="submit"].btn {
...
 
-.cooperation p {margin-bottom: 0px;}
+.cooperation p {
+    margin-bottom: 0px;}
```
Comment (tomasz-biernacki) [inline resources/public/css/style.css]: newline before }

**Pull request merged**